synapseclient/core/utils.py

```diff
@@ -47,6 +50,11 @@
 
 SLASH_PREFIX_REGEX = re.compile(r"\/[A-Za-z]:")
 
+# Set up logging
+LOGGER_NAME = DEFAULT_LOGGER_NAME
```

Comment:
@BryanFauble does this have any impact on the logging for the client?

Reply:
@jaymedina Any changes to the logging should be done in `synapseclient/core/logging_setup.py`

Can you give a small recap of what this change would be accomplishing?

Reply:
My intent here was implementing a logger instance, similar to the way I was it being done in [retry.py](https://github.com/Sage-Bionetworks/synapsePythonClient/blob/1918ec2a667570637fa9c16a459bb55879c1e1d5/synapseclient/core/retry.py#L117). I chose `DEFAULT_LOGGER_NAME` since we are just logging a `Warning` in this case.

Reply:
For this line:
`logging.getLogger("py.warnings").handlers = LOGGER.handlers`

This is changing some of the handlers on the logger instance you are grabbing, could this logic live in `logging_setup.py` instead?

Reply:
Hi @BryanFauble - I did a bit more research on the logging setup and handlers, and realized this line is redundant because I am not using a native python warning, so there's no need to update the handlers with our custom config. I've removed this line and ran another test, and the warning is the same.

```
In [4]: sub = syn.getSubmission("9745366.0")
[WARNING] Submission ID '9745366.0' contains decimals which are not supported. Submission ID will be converted to '9745366'.
```